packages/shared/src/lib/feed.ts

```diff
@@ -120,7 +136,7 @@ export function postAnalyticsEvent(
     target_type: 'post',
     post_type: post.type,
     post_source_type: post.source?.type,
-    extra: opts?.extra ? JSON.stringify(opts.extra) : undefined,
+    extra: Object.keys(extra).length > 0 ? JSON.stringify(extra) : undefined,
```

Comment:
We call this function all over with column and other feed related properties and I tried injecting `extra.feed_id` from there but it was starting to get really messy. 

So I opted into above "hack" to just pull the `feed_id` from current browser URL and inject it. Open for other ideas. FYI @idoshamun